<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kirjautuminen</class>
 <widget class="QDialog" name="kirjautuminen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Kortin tunnusluku:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>260</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="panelButton_2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="panelButton_5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="panelButton_6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="panelButton_0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="panelButton_4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="panelButton_1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="panelButton_7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="panelButton_3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="panelButton_8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="panelButton_9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButtonRESET">
      <property name="text">
       <string>RESET</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSpinBox" name="ekaNumero">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="tokaNumero">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="kolmasNumero">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="neljasNumero">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonRESET</sender>
   <signal>clicked()</signal>
   <receiver>pushButtonRESET</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRESET</sender>
   <signal>clicked()</signal>
   <receiver>ekaNumero</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRESET</sender>
   <signal>clicked()</signal>
   <receiver>tokaNumero</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonRESET</sender>
   <signal>clicked()</signal>
   <receiver>kolmasNumero</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>panelButton_9</sender>
   <signal>clicked()</signal>
   <receiver>neljasNumero</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
